lib/data/kinetic/backend.js

```diff
@@ -0,0 +1,41 @@
+import config from '../../Config';
+
+const backend = {
+    put: function putK(request, size, keyContext, reqUids, callback, drive) {
```

Comment:
Maybe you can track the length of the data received, with a valueLen variable, and then pass it to Buffer.concat() :)